modules/SofaGraphComponent/SceneCheckAPIChange.cpp

```diff
@@ -103,35 +103,30 @@ void SceneCheckAPIChange::doCheckOn(Node* node)
                 hook(object.get());
             }
         }
-
     }
 }
 
 
 void SceneCheckAPIChange::installDefaultChangeSets()
 {
-    addHookInChangeSet("17.06", [](Base* o){
+    addHookInChangeSet("17.06", [this](Base* o){
         if(o->getClassName() == "BoxStiffSpringForceField" )
-            msg_warning(o) << "BoxStiffSpringForceField have changed since 17.06. To use the old behavior you need to set parameter 'forceOldBehavior=true'" ;
-    }) ;
+            msg_warning(this->getName()) << "BoxStiffSpringForceField have changed since 17.06. To use the old behavior you need to set parameter 'forceOldBehavior=true'";
+    });
 
-    addHookInChangeSet("17.06", [](Base* o){
+    addHookInChangeSet("17.06", [this](Base* o){
         if( deprecatedComponents.find( o->getClassName() ) != deprecatedComponents.end() )
         {
-            msg_deprecated(o) << deprecatedComponents.at(o->getClassName()).getMessage();
+            msg_deprecated(this->getName()) << deprecatedComponents.at(o->getClassName()).getMessage();
```

Comment:
This change lead to un-readable message because the name of the component is not showned anymore.